guides/material-2.md

```diff
@@ -714,6 +714,226 @@ You can mirror this structure in your components by defining your own mixins. Th
+easiest way to load Roboto, or any other custom font, is by using Google Fonts. The following
+snippet can be placed in your application's `<head>` to load Roboto from Google Fonts.
+
+```html
+<link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+```
+
+See [Getting Started with the Google Fonts API][fonts-api] for more about using Google Fonts. Also
+note that, by default, [the Angular CLI inlines assets from Google Fonts to reduce render-blocking
+requests][font-inlining].
+
+[roboto]: https://fonts.google.com/share?selection.family=Roboto:wght@300;400;500
+[fonts-api]: https://developers.google.com/fonts/docs/getting_started
+[font-inlining]: https://angular.io/guide/workspace-config#fonts-optimization-options
+
+### Typography levels
+
+A **typography level** is a collection of typographic styles that corresponds to a specific
+part of an application's structure, such as a header. Each level includes styles for font family,
+font weight, font size, and letter spacing. Angular Material uses the [typography levels
+from the 2018 version of the Material Design specification][2018-typography], outlined in the
+table below.
+
+| Name            | Description                                                  |
+|-----------------|--------------------------------------------------------------|
+| `headline-1`     | One-off header, usually at the top of the page (e.g. a hero header). |
+| `headline-2`     | One-off header, usually at the top of the page (e.g. a hero header). |
+| `headline-3`     | One-off header, usually at the top of the page (e.g. a hero header). |
+| `headline-4`     | One-off header, usually at the top of the page (e.g. a hero header). |
+| `headline-5`     | Section heading corresponding to the `<h1>` tag.             |
+| `headline-6`     | Section heading corresponding to the `<h2>` tag.             |
+| `subtitle-1`     | Section heading corresponding to the `<h3>` tag.             |
+| `subtitle-2`     | Section heading corresponding to the `<h4>` tag.             |
+| `body-1`         | Base body text.                                              |
+| `body-2`         | Secondary body text.                                         |
+| `caption`        | Smaller body and hint text.                                  |
+| `button`         | Buttons and anchors.                                         |
+
+[2018-typography]: https://m2.material.io/design/typography/the-type-system.html#type-scale
+
+#### Define a level
+
+You can define a typography level with the `define-typography-level` Sass function. This function
+accepts, in order, CSS values for `font-size`, `line-height`, `font-weight`, `font-family`, and
+`letter-spacing`. You can also specify the parameters by name, as demonstrated in the example below.
+
+```scss
+@use '@angular/material' as mat;
+
+$my-custom-level: mat.define-typography-level(
```

Comment:
I think that some of these were renamed in https://github.com/angular/components/pull/28892 as well.